.changeset/gentle-eagles-care.md

```diff
@@ -0,0 +1,9 @@
+---
+"create-eth": patch
+---
+
+- up hardhat-deploy allowing etherscan v2 api for verification (https://github.com/scaffold-eth/scaffold-eth-2/pull/1116)
+- add not found page (https://github.com/scaffold-eth/scaffold-eth-2/pull/1120)
+- feat: useScaffoldEventHistory `toBlock` param (https://github.com/scaffold-eth/scaffold-eth-2/pull/1117)
```

Comment:
nit: maybe make this line similar to next line and with lowercase? and next line to lowercase too
```
- add `toBlock` param to useScaffoldEventHistory (https://github.com/scaffold-eth/scaffold-eth-2/pull/1117)
- add `blocksBatchSize` param to useScaffoldEventHistory (https://github.com/scaffold-eth/scaffold-eth-2/pull/1118)
```